go.mod

```diff
@@ -0,0 +1,16 @@
+module github.com/segmentio/go-athena
+
+go 1.22
```

Comment:
do we want to heavily restrict the go versions that can consume this package by setting this so high? By setting this, we say that minimum packages must have go 1.22 to consume this, right?

Reply:
I'm generally OK with it. Go 1.21 and older aren't supported any more, so I'm reluctant to go as far back as 1.19: https://endoflife.date/go We could add a tag for the current HEAD, before this commit, for the benefit of current users when they can't move up to a supported Go version.

I also expect to bump this up to aws-sdk-go-v2 in the near future. That's on 1.21 right now, so I'd be OK going back that far.

Reply:
Works for me!